docs/scenarios-workflow.md

```diff
@@ -0,0 +1,152 @@
+  
+
+# Scenario Tests Guide
+
+  
+
+## Overview
+
+  
+
+Our existing scenario tests are under `src\scenarios` in this repo, where each subdirectory contains a test asset that can be combined with a specific set of commands to do measurements. Currently we have scenario tests for [SDK](link), [Crossgen](link), [Blazor](./blazor-scenarios.md) and [other scenarios]().
+
+  
+  
+
+## Running scenario tests
+
+  
+
+This is a general guideline on how the scenario tests are arranged in this repo. We will walk through it by measuring the **startup time of an empty console template** as a sample scenario. For other scenarios, refer to the following links:
+
+  
+
+  
+
+-  [How to run SDK scenario tests](link)
+
+  
+
+-  [How to run Crossgen scenario tests](./crossgen-scenarios.md)
+
+  
+
+-  [How to run Blazor tests](./blazor-scenarios.md)
+
+  
+
+-  [How to run other Scenario tests](link)
+
+  
+### Prerequisites:
+- python3 or newer
+- dotnet runtime 3.0 or newer
+
+### Step 1 Initialize Environment
+
+  
+
+Go to `src\scenarios` and run the following command:
+
+  
+
+On Windows, start a new PowerShell environment ***in Admin Mode*** and run:
+
+  
+
+```
+
+cd src\scenarios
+
+.\init.ps1
+
+```
+
+  
+
+On Linux, run in the terminal:
+
+  
+
+```
+
+cd src/scenarios
+
+. init.sh
+
+```
+
+  
+This script sets up `$PYTHONPATH`. Without other options specified it will use the dotnet in `$PATH`. To download a new dotnet or specify a custom dotnet, type `--help` for more command line options.
```

Comment:
It would be helpful to give example of commonly used parameters, like -Channel, -DotnetDirectory. It's not obvious that what value these parameters expect.

Reply:
It's now included in the script's help message. Can you take a look and let me know if it's understandable?